### ISSUE TITLE
Remove unnecessary css which controls the header width

### ISSUE THREAD

**Issue by viocha** [mugiwara85/CodeblockCustomizer#62]
When the width is enabled, the header do not align with the codeblock.
![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/66580331/7184f8d4-2232-4b99-b53d-ad56b316b776)
When the width is disabed, it gets back to the normal appearance.
![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/66580331/9ce4902e-5584-4729-a272-ef467d4c712d)
The original code is
```css
.markdown-source-view .codeblock-customizer-header-container, .markdown-source-view .codeblock-customizer-header-container-specific {
    width: calc(var(--line-width-adaptive) - var(--folding-offset));
    margin-left: max(calc(50% + var(--folding-offset) - var(--line-width-adaptive)/2), calc(50% + var(--folding-offset) - var(--max-width)/2)) !important;
}
```

**Comment by mugiwara85** [comment]
That's not unnecessary. It has a reason why it is there.
But first of all provide some information. What theme are you using? Why do the icon looks like that? Did you apply some custom css?

**Comment by viocha** [comment]
I'm using the theme "Things" and there is no other custom css.
![image](https://github.com/mugiwara85/CodeblockCustomizer/assets/66580331/a31c68d3-139d-40eb-90cf-ac5b7f2f0835)

**Comment by mugiwara85** [comment]
Ok, I'll look into it, and get back to you

**Comment by mugiwara85** [comment]
Available in current release (1.2.6). Should there be any problems please report back.
